src/server/pfs/server/server.go

```diff
@@ -22,5 +24,21 @@ func NewSidecarAPIServer(env Env) (pfsserver.APIServer, error) {
 	if err != nil {
 		return nil, err
 	}
+	if env.PachwInSidecar {
+		taskSource := env.TaskService.NewSource(StorageTaskNamespace)
+		go compactionWorker(env.BackgroundContext, taskSource, a.driver.storage) //nolint:errcheck
+	}
+	return newValidatedAPIServer(a, env.AuthServer), nil
+}
+
+// NewPachwAPIServer is used for Pachw Mode. Pachw worker pool instances to process background tasks from the task service.
+func NewPachwAPIServer(env Env) (pfsserver.APIServer, error) {
+	a, err := newAPIServer(env)
+	if err != nil {
+		return nil, err
+	}
+	go func() { pfsload.Worker(env.GetPachClient(env.BackgroundContext), env.TaskService) }() //nolint:errcheck
```

Comment:
Shouldn't there be a URL worker here? Also, the above comment reads a bit weird.

Reply:
Good catch